shared_versions.gradle

```diff
@@ -39,5 +39,5 @@ ext {
   tomcatJdbcPoolVersion = '8.5.16'
   tomcatContainerId = 'tomcat8x'
   validationAPIVersion = '1.0.0.GA'
-  predixpublickeyProviderVersion = '1.1.0'
+  predixpublickeyProviderVersion = '1.2.0-SNAPSHOT'
```

Comment:
have we pushed a 1.2.0-SNAPSHOT to artifactory?

Reply:
not yet. Once, the keyProvider changes are ready it will be done